web/app/templates/job.hbs

```diff
@@ -20,86 +20,148 @@
-                    {{tasks-severity tasksseverity=model.tasksseverity}}
-                </td>
-            </tr>
-            <tr>
-                <td id="job_aggregated_metrics">
-                    {{aggregated-metrics application=model}}
-                </td>
-            </tr>
-            </tbody>
-        </table>
-    </div>
-
-    <div class="shadow summary-list-container">
-        <div class="summary-list-header">Applications</div>
-
-        {{application-list applications=model.taskssummaries}}
-    </div>
+  <div class="bread-crumb flat">
+    {{#link-to "dashboard"}}<i class="glyphicon glyphicon-home"></i>{{/link-to}}
+    {{#link-to "workflow" (query-params workflowid=model.jobs.flowexecid)}}Workflow{{/link-to}}
+    {{#link-to "job" (query-params jobid=model.jobs.jobexecid) class="active"}}<label>Job</label>{{/link-to}}
+  </div>
+
+  <div class="shadow details-container box">
+    <h4>{{model.jobs.jobdisplayname}}</h4>
+    <hr class="horizontal-line"/>
+    <table class="info borderless">
+
+
+      <tbody>
+      <tr>
+        <td>User</td>
+        <td id="job_user">{{model.jobs.username}}</td>
+      </tr>
+      <tr>
+        <td>Queue</td>
+        <td id="job_queue">{{model.jobs.queue}}</td>
+      </tr>
+      <tr>
+        <td>Jobtype</td>
+        <td id="job_type">{{model.jobs.jobtype}}</td>
+      </tr>
+      <tr>
+        <td>Start time</td>
+        <td id="job_start_time">{{model.jobs.starttime}}</td>
+      </tr>
+      <tr>
+        <td>End time</td>
+        <td id="job_end_time">{{model.jobs.finishtime}}</td>
+      </tr>
+      </tbody>
+    </table>
+    <table class="info borderless">
+      <tr>
+        <td><a id="job_flow_history" class="btn"
+               href="./newflowhistory?flow-def-id={{urlEncode model.jobs.flowdefid}}&select-graph-type=resources"><span
+            class="glyphicon glyphicon-stats"></span> FlowHistory</a></td>
+        <td><a id="job_flow_def_id" class="btn" href={{model.jobs.flowdefid}}><span
+            class="glyphicon glyphicon-info-sign"></span> Flow
+          definition</a></td>
+        <td><a id="job_flow_exec_id" class="btn" href="{{model.jobs.flowexecid}}"><span
+            class="glyphicon glyphicon-cog"></span> Flow
+          execution</a></td>
+      </tr>
+      <tr>
+        <td><a id="job_job_history" class="btn"
+               href="./newjobhistory?job-def-id={{urlEncode model.jobs.jobdefid}}&select-graph-type=resources"><span
+            class="glyphicon glyphicon-stats"></span> JobHistory</a></td>
+        <td><a id="job_job_def_id" class="btn" href="{{model.jobs.jobdefid}}"><span
+            class="glyphicon glyphicon-info-sign"></span> Job
+          definition</a></td>
+        <td><a id="job_job_exec_id" class="btn" href="{{model.jobs.jobexecid}}"><span
+            class="glyphicon glyphicon-cog"></span> Job
+          execution</a></td>
+      </tr>
+    </table>
+    <table class="info borderless">
+      <tbody>
```

Comment:
Are you dynamically adding columns in this table? If no, using table sounds overkill here.